### PR TITLE
Add a new image of arlas-wui that doesn't embark analytics

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -171,13 +171,18 @@ npm cache clean --force
 rm -rf node_modules/
 
 echo "==> Docker"
-docker build --no-cache --build-arg version=${VERSION} --tag gisaia/arlas-wui:${VERSION} .
+docker build -f docker/Dockerfile-production --no-cache --build-arg version=${VERSION} --tag gisaia/arlas-wui:${VERSION} .
+docker build -f docker/Dockerfile-production-no-analytics --no-cache --build-arg version=${VERSION} --tag gisaia/arlas-wui-no-analytics:${VERSION} .
```
Comment (sebbousquet) [inline release.sh:175]: I think we should declare a version with no analytics, not another docker

`--tag gisaia/arlas-wui:${VERSION}-no-analytics`

File: release.sh [hunk 2]
```diff
@@ -171,13 +171,18 @@ npm cache clean --force
...
 
 docker push gisaia/arlas-wui:${VERSION}
+docker push gisaia/arlas-wui-no-analytics:${VERSION}
```
Comment (sebbousquet) [inline release.sh:178]: Same here => `docker push gisaia/arlas-wui:${VERSION}-no-analytics`

File: release.sh [hunk 3]
```diff
@@ -171,13 +171,18 @@ npm cache clean --force
...
 if [ "${STAGE}" == "stable" ] && [ "${IS_LATEST_VERSION}" == "YES" ];
     then
     docker tag gisaia/arlas-wui:${VERSION} gisaia/arlas-wui:latest
     docker push gisaia/arlas-wui:latest
+
+    docker tag gisaia/arlas-wui-no-analytics:${VERSION} gisaia/arlas-wui-no-analytics:latest
+    docker push gisaia/arlas-wui-no-analytics:latest
```
Comment (sebbousquet) [inline release.sh:185]: Remove this part, `latest` is regular version of `arlas-wui`
Version with no-analytics is specific

**Review by mbarbet — APPROVED** [review]

File: release.sh (hunk 3 above)
Comment (MohamedHamouGisaia) [reply release.sh]: fixed

File: release.sh (hunk 1 above)
Comment (MohamedHamouGisaia) [reply release.sh]: fixed

File: release.sh (hunk 2 above)
Comment (MohamedHamouGisaia) [reply release.sh]: fixed

**Review by sebbousquet — APPROVED** [review]

**Review by QuCMGisaia — APPROVED** [review]

**Review by mbarbet — APPROVED** [review]

**Pull request merged**